src/middlewared/middlewared/plugins/cloud_sync.py

```diff
@@ -218,7 +201,7 @@ class Config:
     def _validate(self, schema_name, data):
         verrors = ValidationErrors()
 
-        if data["provider"] not in REMOTES:
+        if data.get("provider") not in REMOTES:
```

Comment:
`None` would never be in `REMOTES`, why?

Reply:
Reason I used get was that we are not enforcing that provider has to be provided in do_create Dict, raising a possibility of a  KeyError. But I totally missed what the verror was for. I think I should just make provider required then ? Thoughts @themylogin ?

Reply:
@sonicaj I've totally missed my own plugin when I was fixing errors like that :D Thank you for pointing that, there are a few errors like that in that plugin, I'll fix them.

Reply:
No problem : ) I'll change it back to data['provider'] then, Thank you : )